src/Config/Constants.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Constants Name of Shield Tables
+ */
+/** @var \CodeIgniter\Shield\Config\Auth authConfig */
```

Comment:
Sorry, there was no `$`.
```suggestion
/** @var \CodeIgniter\Shield\Config\Auth $authConfig */
```